package.json

```
@@ -30,8 +30,8 @@
     "eslint-config-airbnb": "^5.0.1",
     "eslint-plugin-react": "^3.16.1",
     "jest": "^18.1.0",
+    "prettier": "^1.18.2",
```

```suggestion
    "prettier": "1.18.2",
```

I recommend enforcing a specific version of prettier to avoid formatting diffs until the version gets intentionally bumped up.